src/js/containers/search/filters/KeywordContainer.jsx

```diff
@@ -20,18 +20,10 @@ const ga = require('react-ga');
 
 export class KeywordContainer extends React.Component {
 
-    static logFilterEvent() {
-        ga.event({
-            category: 'Search Filters',
-            action: 'Applied Filter',
-            label: 'Keyword'
-        });
-    }
-
     static logSelectedKeywordEvent(keyword) {
         ga.event({
-            category: 'Search Filters',
-            action: 'Selected Keyword',
+            category: 'Search Page Filter Applied',
+            action: 'Applied Keyword Filter',
             label: keyword
```

Comment:
If I clear the box and press Submit, we'll be logging an empty string. Should we check if the keyword isn't null?